schemas/json/layout/footer.schema.v1.json

```diff
@@ -0,0 +1,82 @@
+{
+  "$id": "https://altinncdn.no/schemas/json/layout/footer.schema.v1.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Altinn footer",
+  "description": "Schema that describes the footer configuration for Altinn applications.",
+  "type": "object",
+  "additionalProperties": true,
```

Comment:
Personally, I'd like for us to start disallowing additional properties everywhere. Our form layout definition allows additional properties, and it has made it difficult to notice that we forgot to add schema definitions for some new property. IMO, if you add some value to the footer schema that we never expected (and will not handle in any way), that should also present you with an error in vscode.